src/m365/flow/commands/flow-list.ts

```diff
@@ -128,26 +138,28 @@ class FlowListCommand extends PowerAutomateCommand {
         index === self.findIndex(f => f.id === flow.id)
       );
 
-      if (items.length > 0) {
-        items.forEach(i => {
-          i.displayName = i.properties.displayName;
+      if (args.options.output && args.options.output !== 'json') {
+        items.forEach(flow => {
+          flow.displayName = flow.properties.displayName;
         });
-
-        await logger.log(items);
-      }
-      else {
-        if (this.verbose) {
-          await logger.logToStderr('No Flows found');
-        }
       }
+      await logger.log(items);
     }
     catch (err: any) {
       this.handleRejectedODataJsonPromise(err);
     }
   }
 
   private getApiUrl(environmentName: string, asAdmin?: boolean, includeSolutionFlows?: boolean, filter?: 'personal' | 'team',): string {
```

Comment:
I think we can simplify this method a bit. I was thinking something along the lines of the following:

``` typescript
    const baseEndpoint = `${this.resource}/providers/Microsoft.ProcessSimple`;
    const environmentSegment = `/environments/${formatting.encodeQueryParameter(environmentName)}`;
    const adminSegment = `/scopes/admin${environmentSegment}/v2`;
    const flowsEndpoint = '/flows?api-version=2016-11-01';
    const filterQuery = filter === 'personal' || filter === 'team' ? `&$filter=search('${filter}')` : '';
    const includeQuery = includeSolutionFlows ? '&include=includeSolutionCloudFlows' : '';

    return `${baseEndpoint}${asAdmin ? adminSegment : environmentSegment}${flowsEndpoint}${filterQuery}${includeQuery}`;
```